### DISCUSSION TITLE
Aggregating on postgres interval columns

### DISCUSSION THREAD

**Discussion by otonnesen** [hasura/graphql-engine#9365] [Question] [answered]
Hi all,

I'm trying to perform an aggregation query on a table with an interval column, but I'm getting an error. I couldn't find any info on interval aggregations in the docs github issues. Am I just missing something obvious in the example below, or can intervals not be used in Hasura's built-in aggregation utilities?

### Context

Running Postgres 14.4 and Hasura 2.16.1.

Table:
```sql
create table test (id bigint, duration interval);
```
Query:
```gql
{
  test_aggregate {
    aggregate {
      sum {
        duration
      }
    }
    nodes {
      id
    }
  }
}
```
Error:
```json
{
  "errors": [
    {
      "extensions": {
        "code": "validation-failed",
        "path": "$.selectionSet.test_aggregate.selectionSet.aggregate.selectionSet.sum.selectionSet.duration"
      },
      "message": "field 'duration' not found in type: 'test_sum_fields'"
    }
  ]
}
```

**Comment by adas98012** [comment, marked as answer]
@otonnesen 
We recognize this as a feature request. You'll be able to follow its status [here](https://github.com/hasura/graphql-engine/issues/9375)

**Reply by otonnesen** [reply]
Perfect, thanks!
